tests/integration/docusaurus/tutorials/quickstart/quickstart.py

```diff
@@ -31,6 +31,13 @@
 )
 # </snippet>
 
+# Not sure if this is necessary for the 'default' expectation_suite_name, but
+# great_expectations.data_context.abstract_data_context.get_expectation_suite()
+# (when called during "checkpoint.run()" below will complain that expectation_suite
+# 'default' is not found
+suite_name = checkpoint.get_config().expectation_suite_name
+context.add_or_update_expectation_suite(expectation_suite_name=suite_name)
```

Comment:
I'm surprised - is this a known issue? cc @NathanFarmer

Reply:
Yea that seems weird.

Reply:
I can't reproduce this locally. I'm not sure what's going on here, but I just noticed we never call `save_expectation_suite()` in the quickstart and I'm wondering if it ever gets called under the hood.

Reply:
"Complains" means a warning?

Reply:
I'll take it out and re-run locally

Reply:
Seems ok now without it.